lxd/project/permissions.go

```diff
@@ -1244,7 +1244,7 @@ func expandInstancesConfigAndDevices(instances []api.Instance, profiles []api.Pr
 		}
 
 		expandedInstances[i] = instance
-		expandedInstances[i].Config = db.ExpandInstanceConfig(instance.Config, apiProfiles)
+		expandedInstances[i].Config = db.ExpandInstanceConfig(nil, nil, instance.Config, apiProfiles)
```

Comment:
I think we should add a comment explaining why we arent passing in global config here.